### PR TITLE
Add postsubmits jobs for coverage…

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

… mimics knative/test-infra on this.

It should help for https://github.com/tektoncd/pipeline/issues/988

/cc @bobcatfish 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ :no_good_man: ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: prow/config.yaml [hunk 1]
```diff
@@ -631,6 +631,21 @@ presubmits:
...
+    decorate: true
+    spec:
+      containers:
+      - image: gcr.io/knative-tests/test-infra/coverage:latest # we may want ours here
+        imagePullPolicy: Always
+        command:
+        - "/coverage"
+        args:
+        - "--artifacts=$(ARTIFACTS)"
+        - "--cov-threshold-percentage=0"
```
Comment (houshengbo) [inline prow/config.yaml]: @vdemeester Is there any particular reason that we set 0 percent for post-tekton jobs?

Comment (vdemeester) [reply prow/config.yaml]: as they are postsubmit (aka once it's merged), doesn't really make sense to make them fail

**Comment by vdemeester** [comment]
It is deployed :angel: 
cc @abayer

**Pull request merged**